metamodel.zenta

```diff
@@ -1496,28 +1481,36 @@ count(zenta:neighbours($p1,$p2,'uses,1')[@xsi:type='HTTP method']) = 1</document
             <property key="font"/>
           </element>
         </folder>
-        <folder name="06. Repository" id="0af3528f-c853-4ded-b956-9e5197174d9e">
+        <folder name="07. Repository" id="0af3528f-c853-4ded-b956-9e5197174d9e">
```

Comment:
Repository should be joined to Entity type

Reply:
ACK